src/main/resources/find-elements-by-text-case-insensitive.js

```diff
@@ -1,18 +1,9 @@
 (function () {
+  import 'texts.js'
```

Comment:
I'd really prefer not to use the word `import` here to avoid any ambiguity with `ESM modules in JS`. Any other word that conveys we're adding text from another file would be better, to make it clear that this is not an ESM import at all.

embed\appead sounds good.

Reply:
What I like about `import` is that IDEA shows it without highlighting syntax errors, and allows to navigate into "texts.js" by clicking it.

<img width="543" alt="image" src="https://github.com/user-attachments/assets/e13e68c7-fe9a-4afe-876a-31206cb52ab6">

If I replace `import` by `embed`, IDEA highlights it as an error, and doesn't allow navigation by click:

<img width="517" alt="image" src="https://github.com/user-attachments/assets/96ae6559-ddd1-45ca-a9e1-d105c42bf007">